src/testkit/scala/tools/testkit/AssertUtil.scala

```diff
@@ -201,9 +201,41 @@ object AssertUtil {
 
   /** Like Await.ready but return false on timeout, true on completion, throw InterruptedException. */
   def readyOrNot(awaitable: Awaitable[_]): Boolean = Try(Await.ready(awaitable, TestDuration.Standard)).isSuccess
+
+  /** Syntax to clarify which is the expected value. */
+  def assertEqualTo[A](expected: A)(actual: A): Unit = assertEquals(expected, actual)
```

Comment:
This cries for a shiny new extension method: `(1 + 1) assertEqualTo 2`...

Reply:
Don't make me dig out that meme with the red dress.

We recently re-watched The Matrix, which I don't adore as some do, but the daughter said we watched it already, and then oh that's where that meme comes from, meaning the red/blue pill. I can never remember which pill is which. Anyway, the red dress from Matrix is a thing but I think not a meme? This red dress is the distraction that leads to destruction. It was conceived as a test, but the coder guy says he has recycled it for other temptations.